.eslintrc.json

```diff
@@ -18,7 +18,8 @@
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "react/react-in-jsx-scope": "off",
     "import/no-unresolved": "off",
-    "no-shadow": "off"
+    "no-shadow": "off",
+    "react/no-unused-prop-types": "off"
   },
   "ignorePatterns": [
     "dist/",
```

Comment:
### Error
![error](https://user-images.githubusercontent.com/93039359/186426390-68ea7824-c46e-442f-b484-c52f07e13d5e.png)

- [ ] We are getting this error after starting your application please fix it :+1: